### PR TITLE
Descriptive text

### PR DESCRIPTION
**Pull request by mtslaugh**
Add arbitrary text

### CODE REVIEW

**Comment by ilankham** [comment]
Sorry, I can't take this PR right now.

**Comment by ilankham** [comment]
Maybe try again now?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # test-repo
 
 R & Python are best friends
+
+Arbitrary text to be rejected
```
Comment (ilankham) [inline README.md:5]: This text seems too arbitrary to me.

**Review by ilankham — COMMENTED** [review]
Please correct and @-me when ready for another review of this PR.

File: README.md (hunk 1 above)
Comment (mtslaugh) [reply README.md]: Oh, sorry, hopefully this is better!

**Review by ilankham — APPROVED** [review]
LGTM!

**Pull request merged**